site/_data/i18n/tags.yml

```diff
@@ -23,6 +23,34 @@ autoplay:
 capabilities:
   en: Capabilities
 
+# DevTools-related tags
+css:
+  en: CSS
+
+javascript:
+  en: JavaScript
+
+get-started:
+  en: Get started with DevTools
```

Comment:
It would be great to make this a bit more generic -- I wouldn't be surprised if other blog posts end up using "Get started" that aren't related to DevTools